docs/attributes-registry/vcs.md

```diff
@@ -29,6 +31,14 @@ it is identical to the `ref.name`, it SHOULD still be included. It is
 up to the implementer to decide which value to set as the revision
 based on the VCS system and situational context.
 
+`vcs.change.state` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value    | Description                                                                                    | Stability                                                        |
+| -------- | ---------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `closed` | closed                                                                                         | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `merged` | [merged](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddeftagatag)     | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `open`   | [open](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefbranchabranch) | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
These links are a copy/paste mistake.

Do you have any good definitions I could link to?

If not, I could replace the links with actual descriptions.

Reply:
I could not find any good definitions for these states on existing webpages.
In afaa1f5bffd0d548e50ddb664eff5175723d2879 I added definitions given by Copilot based on the github, gitlab and microsoft documentation.
I adapted the definitions for `vcs.change.state`.